File: src/main/java/org/ventanas/Java/Ejercicio1.java
package org.ventanas.Java;
//1. Saludo personalizado
// Pedir al usuario su nombre y saludarle con un mensaje personalizado. Ejemplo: "¡Hola,
// Juan!".
import java.util.Scanner;

public class Ejercicio1 {
    public static void execute() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Como te llamas: ");
        String nombre = scanner.nextLine();
        System.out.println("¡Hola, " + nombre + "!");
    }
}

File: src/main/java/org/ventanas/Kotlin/Ejercicio1.kt
package org.ventanas.Kotlin
//1. Saludo personalizado
// Pedir al usuario su nombre y saludarle con un mensaje personalizado. Ejemplo: "¡Hola,
// Juan!".

fun ejercicio1() {
    print("Como te llamas: ")
    val nombre = readln()
    println("¡Hola, $nombre!")
}